linera-views/src/memory.rs

```diff
@@ -63,17 +68,20 @@ pub struct MemoryStore {
     max_stream_queries: usize,
     /// The namespace of the store
     namespace: String,
+    /// The root_key of the store
```

Comment:
```suggestion
    /// The root key of the store
```

Or "The key of the root view"?

Reply:
Ok. I do not want to use the "root view" as it may collides with the notion of view.